From bec88f4909887605c85df0313c074e6d9bc25b45 Mon Sep 17 00:00:00 2001
From: Mike Sarahan <msarahan@gmail.com>
Date: Fri, 25 Jan 2019 10:09:44 -0600
Subject: [PATCH] add LIBRARY_LIB path to building py ext on win

---
 python/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index d78fb77..330157c 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -67,13 +67,13 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/src/pycryptosat.cpp.in" "${CMAKE_CUR
 if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     add_custom_command(OUTPUT ${OUTPUT}/timestamp
        COMMAND ${PYTHON_EXECUTABLE}
-       ARGS setup.py build_ext --inplace
+       ARGS setup.py build_ext -L $ENV{LIBRARY_LIB}
        DEPENDS libcryptominisat5
    )
 else()
     add_custom_command(OUTPUT ${OUTPUT}/timestamp
        COMMAND ${PYTHON_EXECUTABLE}
-       ARGS setup.py build_ext --inplace --rpath ../lib
+       ARGS setup.py build_ext --rpath ../lib
        DEPENDS libcryptominisat5
    )
 endif()
-- 
2.17.1

